### ISSUE TITLE
Add metric in model compile

### ISSUE THREAD

**Issue by jfmwz** [natalinobusa/deepchurn#1]
Change the defintion of model.compile to

model.compile(loss='binary_crossentropy', optimizer='adam',  metrics=['accuracy'] )

Because if not, you only get the lost in score variable
